_data/site.json

```diff
@@ -30,6 +30,10 @@
         {
             "label": "Blog",
             "url": "/blog/"
+        },
+		        {
+            "label": "Jobs",
+            "url": "/jobs/"
```

Comment:
I guess Bounties is an awkward word to use?

Jobs makes it sound like we're hiring employees.

Reply:
Yeah at first I thought that it would be more generic this way, but it's indeed misleading.